tests/testthat/helper.R

```diff
@@ -0,0 +1,8 @@
+if (gargle:::secret_can_decrypt("gcalendr")) {
+  json <- gargle:::secret_read("gcalendr", "gcalendr-testing.json")
```

Comment:
We need to actually put an encrypted service account token in the expected location for this to work. Namely at `inst/secret/gcalendr-testing.json`. We will do that like so:

```
gargle:::secret_write(
  package = "gcalendr",
  name = "gcalendr-testing.json",
  input = "a/very/private/local/folder/gcalendr-testing.json"
)
```

after we download a service account token to `a/very/private/local/folder/gcalendr-testing.json`.

Excerpting the important bits of https://gargle.r-lib.org/articles/articles/managing-tokens-securely.html